api/link_muslc.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux && muslc
```

Comment:
I don't know why I get this change, but `go fmt ./...` created it

Reply:
I have seen these pop up, seems like the new go directives...